docs/specifications/cva6_requirement_specification.adoc

```diff
@@ -0,0 +1,391 @@
+
+[RVpriv]  “The RISC-V Instruction Set Manual, Volume II: Privileged Architecture, Document Version  20211203”,  Editors  Andrew  Waterman, Krste  Asanović and John Hauser,  RISC-V Foundation, December 4, 2021.
+
+[RVdbg] “RISC-V External Debug Support, Document Version 0.13.2”, Editors Tim Newsome and Megan Wachs, RISC-V Foundation, March 22, 2019.
+
+[RVcompat] “RISC-V Architectural Compatibility Test Framework”, https://github.com/riscv-non-isa/riscv-arch-test[https://github.com/riscv-non-isa/riscv-arch-test]. +
+
+[AXI] AXI Specification, https://developer.arm.com/documentation/ihi0022/hc[https://developer.arm.com/documentation/ihi0022/hc]. +
+
+[CV-X-IF] Placeholder for the CV-X-IF coprocessor interface currently prepared at OpenHW Group; current version in https://docs.openhwgroup.org/projects/openhw-group-core-v-xif/[https://docs.openhwgroup.org/projects/openhw-group-core-v-xif/]. +
+
+[OpenPiton] “OpenPiton Microarchitecture Specification”, Princeton University, https://parallel.princeton.edu/openpiton/docs/micro_arch.pdf[https://parallel.princeton.edu/openpiton/docs/micro_arch.pdf]. +
+
+== Reference specifications
+
+[RVcmo] “RISC-V Base Cache Management Operation ISA Extensions, version 1.0-fd39d01, 2022-01-12” +
+
+[CLINT] Core-Local Interruptor (CLINT), “SiFive E31 Core Complex Manual v2p0”, chapter 6, https://static.dev.sifive.com/SiFive-E31-Manual-v2p0.pdf[https://static.dev.sifive.com/SiFive-E31-Manual-v2p0.pdf] + 
+
+== Other documents
+
+[FPGAreset] Ken Chapman, “Get Smart About Reset: Think Local, Not Global”, Xilinx WP272 white paper, https://www.xilinx.com/support/documentation/white_papers/wp272.pdf[https://www.xilinx.com/support/documentation/white_papers/wp272.pdf]. +
+
+== Functional requirements
+
+=== General requirement
+
+|===
+
+|GEN&#8209;10|CVA6 shall be *fully compliant with RISC-V specifications* [RVunpriv], [RVpriv] and [RVdbg] by implementing all mandatory features for the set of extensions that are selected and by passing [RVcompat] compatibility tests.
+
+|===
+
+As the RISC-V specification leaves space for options, this specification is a place where to specify these options.
+
+=== RISC-V standard instructions
+
+To ease tracing to verification, the extensions have been split in independent requirements.
+|===
+
+|ISA&#8209;10|CV64A6 shall support *RV64I* base instruction set, version 2.1.
+|ISA&#8209;20|CV32A6 shall support *RV32I* base instruction set, version 2.1.
+|ISA&#8209;30|CVA6 shall support the **M** extension (integer multiply and divide), version 2.0.
+|ISA&#8209;40|CVA6 shall support the **A** extension (atomic instructions), version 2.1.
+|ISA&#8209;50|CV32A6 shall support as an *option* the **F** extension (single-precision floating-point), version 2.2.
+|ISA&#8209;60|CV64A6 shall support as an *option* the **F** and **D** extensions (single- and double-precision floating-point), version 2.2.
```

Comment:
Since F extension is specified by ISA-70 why is it also listed in ISA-60?

Reply:
CV64A6 cannot support the D extension with the F extension. I have added a note after the table so that it is clearer.